ui/src/app/job-details/common/debug-icons/debug-icons.component.ts

```diff
@@ -29,6 +29,8 @@ export class JobDebugIconsComponent implements OnInit {
   logFileData: Map<string, string> = new Map();
   private readonly canGetFileContents:boolean;
   private readonly capabilities: CapabilitiesResponse;
+  private readonly gcpBatchOperationIdRegex = 
+    new RegExp("^projects/\(?<projectId>[^/]+\)/locations/\(?<location>[^/]+\)/jobs/job-\(?<batchJobId>[-a-fA-F0-9]+\)$", "i");
```

Comment:
just out of curiosity, what does a PAPI operation id look like? (that wouldn't fall into this regex)

Reply:
Life Sciences variously used the term "pipelines" and "operations", but never "jobs".
```
projects/983594634584/locations/us-central1/operations/2587741339420326006
```